core/src/main/resources/hudson/util/HudsonIsRestarting/index.jelly

```diff
@@ -56,6 +56,17 @@ THE SOFTWARE.
                         <p class="restarting">
                             ${%Your browser will reload automatically when Jenkins is ready.}
                         </p>
+                        <j:if test="${it.safeRestart}">
+                            <div class="safe-restarting">
+                                <p>
+                                    <strong>${%Safe Restart}</strong>
+                                </p>
+                                <p>
+                                    ${%Jobs on agents can usually continue.}
```

Comment:
Best to have a neutral placeholder and a corresponding "English" translation in a resource file for anything nontrivial (more than 1-2 words).

Also, builds run on agents, not jobs.

Reply:
Will Jenkins default to the english translation if there is none for the user-configured language?

Reply:
Yes, there's plenty of `${%blurb}` and similar in the view definitions :)